### PR TITLE
update link on documentation

### LINKED ISSUE #1134: Remove unused link from document
Remove internal web page link from this page: https://imap-processing.readthedocs.io/en/latest/development-guide/getting-started.html

Then add additional changes as needed

### PR DESCRIPTION
**Pull request by lacoak21**
# Change Summary

closes #1134

## Overview
Update the "other" documentation in the getting started page. 

## Updated Files
- docs/source/development-guide/getting-started.rst
   - Removed the link to the galaxy help page
   - Updated wording to be more clear.

### CODE REVIEW

File: docs/source/development-guide/getting-started.rst [hunk 1]
```diff
@@ -28,8 +28,7 @@ Our tests are run using pytest:
     poetry shell
     pytest
 
-Other Information
+Related Information
```
Comment (nkerman) [inline docs/source/development-guide/getting-started.rst:31]: Great change!

File: docs/source/development-guide/getting-started.rst [hunk 2]
```diff
@@ -28,8 +28,7 @@ Our tests are run using pytest:
...
 -----------------
 
-Another source of useful links is the `internal IMAP help page <https://lasp.colorado.edu/galaxy/pages/viewpage.action?pageId=193401231>`_, or the
-`infrastructure documentation <https://sds-data-manager.readthedocs.io/en/latest/>`_.
\ No newline at end of file
+For more information relating to infrastructure, see `sds-data-manager <https://sds-data-manager.readthedocs.io/en/latest/>`_.
```
Comment (nkerman) [inline docs/source/development-guide/getting-started.rst:34]: Looks good.

**Review by nkerman — APPROVED** [review]
This changes everything. Absolutely flawless code.

**Review by torimarbois — APPROVED** [review]
flawless

**Review by tech3371 — APPROVED** [review]
First PR!

**Pull request merged**